modules/nf-core/fastp/tests/main.nf.test

```diff
@@ -19,11 +19,10 @@ nextflow_process {
                 save_trimmed_fail = false
                 save_merged       = false
 
-                input[0] = [
+                input[0] = Channel.of([
```

Comment:
Is `Channel.of` necessary? 
I usually wouldn't care, but since this will be propagated to a lot of modules, I'd try to keep it as minimal as possible.

Reply:
No, but I'd rather be explicit